hs_access_control/migrations/0002_auto_20150817_1150.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.contrib.auth.models import User
+from hs_access_control.models import UserAccess
+
+
+def migrate_users(apps, schema_editor):
+    # create a 'UserAccess' record for each existing user - needed for the new access control to work
+    UserAccess.objects.all().delete()
+    for u in User.objects.all():
+        ua = UserAccess(user=u, admin=u.is_superuser)
```

Comment:
This is more a question for @alva and @dtarb in terms of behavior requirements. Why do we need two different admin flags for each user?